src/content/en/updates/2018/12/get-installed-related-apps.md

```diff
@@ -146,12 +142,12 @@ your app on that platform.
 The `url` property is optional, and the API works fine without it. On Android,
 the `platform` must be `play`. On other devices, `platform` will be different.
 
-#### Define the relationship to your web app {: #relationship-native }
+### Define the relationship to your web app {: #relationship-native }
 
 Each platform has its own method of verifying a relationship. On Android, the
-[Digital Asset Links system](/digital-asset-links/v1/getting-started) is used
-to define the association between a website and an application. On other
-platforms, the way you define the relationship will differ slightly.
+[Digital Asset Links system](/digital-asset-links/v1/getting-started) defines
+the association between a website and an application. On other platforms, the
+way you define the relationship will differ slightly.
```

Comment:
'On other platforms'

We don't need to do it now, but we probably need to clarify this. I assume iOS will support this some time between the next millennium and never. As I recall, MS is interested in supporting this on Windows.